cylc/uiserver/app.py

```diff
@@ -0,0 +1,343 @@
+#!/usr/bin/env python3
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+from pathlib import Path, PurePath
+import sys
+from typing import List
+
+from pkg_resources import parse_version
+from tornado import ioloop
+from tornado.web import RedirectHandler
+from traitlets import (
+    Float,
+    TraitError,
+    TraitType,
+    Undefined,
+    Unicode,
+    default,
+    validate,
+)
+
+from jupyter_server.extension.application import ExtensionApp
+
+from cylc.flow.cfgspec.globalcfg import GlobalConfig
+from cylc.flow.network.graphql import (
+    CylcGraphQLBackend, IgnoreFieldMiddleware
+)
+
+from cylc.uiserver import (
+    __file__ as uis_pkg
+)
+from cylc.uiserver.data_store_mgr import DataStoreMgr
+from cylc.uiserver.handlers import (
+    CylcStaticHandler,
+    CylcVersionHandler,
```

Comment:
These two handlers are not exported in `cylc.uiserver.handlers`. Not too important, we can just ignore here, update the `__all__`, or drop that.

Reply:
Removed the `__all__`.